scripts/bash-completion/bpftrace

```diff
@@ -0,0 +1,91 @@
+# bash completion of bpftrace
+
+_bpftrace_filedir()
+{
+  # bash-completion 2.11-1091-g6e8a1546bde2 rename _filedir to _comp_compgen_filedir
+  if [[ ${BASH_COMPLETION_VERSINFO[0]} == 2 ]] && \
+     [[ ${BASH_COMPLETION_VERSINFO[1]} > 11 ]]; then
+    _comp_compgen_filedir "${@}"
+  else
+    _filedir "${@}"
+  fi
+}
+
+_bpftrace()
+{
+  local cur prev words
+
+  _init_completion -- "$@" || return
+
+  local all_args='-B -f -o -e -h --help -I --include -l -p -c
+                  --usdt-file-activation --unsafe -q --info -k -kk
+                  -V --version --no-warnings -v --dry-run -d
+                  --emit-elf --emit-llvm'
+
+  case ${prev} in
+  -B)
+    COMPREPLY=( $(compgen -W "line full none" -- ${cur}) )
+    return 0
+    ;;
+  -f)
+    COMPREPLY=( $(compgen -W "text json" -- ${cur}) )
+    return 0
+    ;;
+  -c)
+    COMPREPLY=( $(compgen -c -- ${cur}) )
+    return 0
+    ;;
+  -o | -I | --include | --emit-elf | --emit-llvm)
+    _bpftrace_filedir
+    return 0
+    ;;
+  -I)
+    _bpftrace_filedir -d
+    return 0
+    ;;
+  -e)
+    # TODO: or replace '${PROBES} ${ITERS}' to '$(sudo bpftrace -l)', however,
+    # `sudo' maybe need password, and 'bpftrace -l' is too slow, could be a
+    # better way?
+    local PROBES="BEGIN END kprobe kretprobe tracepoint rawtracepoint fentry fexit uprobe uretprobe usdt"
+    local ITERS="iter:bpf_link
+                 iter:bpf_map
+                 iter:bpf_map_elem
+                 iter:bpf_prog
+                 iter:bpf_sk_storage_map
+                 iter:cgroup
+                 iter:ipv6_route
+                 iter:ksym
+                 iter:netlink
+                 iter:sockmap
+                 iter:task
+                 iter:task_file
+                 iter:task_vma
+                 iter:tcp
+                 iter:udp
+                 iter:unix"
+    local MISCS="hardware software interval"
+    COMPREPLY=( $(compgen -W "${PROBES} ${ITERS} ${MISCS}" -- ${cur}) )
+    return 0
```

Comment:
Thinking about this again, I think trying to complete contents of `-e`  causes more problems than it solves. One problem is keeping the list here in sync.

Another practical one is if you type `bpftrace -e "inte<TAB>` you get completed to `bpftrace -e "interval"`. Which is not quite right. And then it only works for the first probe.

So IMO it's better to just drop `-e` content completion.

Reply:
just drop -e content completion, please review, thanks!